boards/arm/nrf5340_audio_dk_nrf5340/nrf5340_audio_dk_nrf5340_cpuapp_common.dts

```diff
@@ -64,6 +66,50 @@
 	pinctrl-0 = <&i2c1_default>;
 	pinctrl-1 = <&i2c1_sleep>;
 	pinctrl-names = "default", "sleep";
+
+	vbat_sensor: ina231@44 {
+		compatible = "ti,ina230";
```

Comment:
the "ina230" name is from when the driver was written with INA230 in mind right? Should be changed to ina23x

Reply:
Same with INA230_CONFIG